packages/render-fidelity-tools/src/components/renderers/babylon-viewer.ts

```diff
@@ -114,17 +114,18 @@ export class BabylonViewer extends LitElement {
 
     this[$scene].stopAllAnimations();
 
-    // load hdr directly (the size of cubmap is set to be 256 for all renderers)
+    // load and prefilter HDR texture (the size of cubmap is set to be 256 for
+    // all renderers)
     const environment = new HDRCubeTexture(
-        scenario.lighting, this[$scene], 256, false, false, false);
+        scenario.lighting, this[$scene], 256, false, false, false, true);
     this[$scene].environmentTexture = environment;
-    // rotate both skybox and hdr texture for 180 deg to match other renderers
+    // rotate environment for 90 deg, skybox for 270 deg to match the rotation
+    // in other renderers
```

Comment:
Babylonjs seems to change their default orbit in their latest version.

Reply:
Wow, you have to set the environment and skybox to different orientations to get it to match? That sounds like a bug; I'd recommend you'd file something on Babylon for that.

Reply:
Ok, i will do that after this pr is merged